docs/content/user-guides/cert-manager.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Traefik Cert Manager Integration"
+description: "Learn how to configure Traefik Proxy to use Cert Manager for certificate on your routers. Read the technical documentation."
+---
+
+# Cert Manager
+
+Provision TLS Certificate for Traefik Proxy with Cert Manager on Kubernetes
+{: .subtitle }
+
+You can configure Traefik Proxy to use Certificates of Cert Manager on Kubernetes.
+
+## Checklist
+
+To obtain a certificate from Cert Manager, you'll need to:
+
+1. Deploy Cert Manager
+2. Configure an Issuer (or a ClusterIssuer)
+3. Create a Certificate with this Issuer
+4. Deploy Traefik Proxy
```

Comment:
I am not sure if this step should be added to the list as it can be misleading, someone can think that Traefik Proxy has to be deployed after Cert-Manager, it is more a prerequisite.

Reply:
Agreed.

Reply:
@kevinpollet After second thought, yes it can be misleading.
@jsoref I have followed your suggestion. 

Thanks !